external/tools/tools.depproj

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <NugetRuntimeIdentifier>$(ToolRuntimeRID)</NugetRuntimeIdentifier>
```

Comment:
Had to add a new `tools.depproj` so that `Ilasm` and `Ildasm` could be restored cross-plat using the `ToolRuntimeRID`

Reply:
Ideally we would do that, but there are a number of "issues" that arise due to the way CoreFX does restore today.
For example, there is no proper intermediate folder for the various `*.nuget.g.props/targets` and they all get spilled next to the ilproj (or csproj) file.

This is what easily works with the current build system. Calling `dotnet restore && dotnet build` directly on the ilproj works, but doesn't plugin to the existing `build.cmd` infrastructure.

Reply:
OK can you please file a follow-up issue on this to fix once we finish our move to arcade?